app/src/main/java/org/fossasia/susi/ai/device/DeviceActivity.kt

```diff
@@ -51,23 +51,18 @@ class DeviceActivity : AppCompatActivity(), IDeviceView {
     }
 
     fun chooseWifi(view: View) {
-        val builder = AlertDialog.Builder(this)
-
-        builder.setTitle(R.string.device_hosted_connection)
-        builder.setMessage(R.string.choose_wifi)
-        builder.setPositiveButton(R.string.ok) { dialog, which ->
-            val intent = Intent(Intent.ACTION_MAIN, null)
-            intent.addCategory(Intent.CATEGORY_LAUNCHER)
-            val componentName = ComponentName("com.android.settings", "com.android.settings.wifi.WifiSettings")
-            intent.component = componentName
-            intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK
-            startActivity(intent)
-        }
 
-        builder.setNegativeButton(R.string.cancel) { dialog, which ->
-            dialog.dismiss()
-        }
+        noDeviceFound.visibility = View.GONE
+        deviceTutorial.visibility = View.GONE
+
+        Handler().postDelayed({
+            scanDevice.visibility = View.GONE
+            scanProgress.visibility = View.GONE
+            noDeviceFound.visibility = View.VISIBLE
+            deviceTutorial.visibility = View.VISIBLE
+        }, 10000)
```

Comment:
Why this?

Reply:
@iamareebjamal this is currently just a mock of how UI would change once the app starts scanning the devices.

Reply:
Why wait 10 seconds

Reply:
it is the timeout as similar to the Google Home app